src/components/tweetCard.vue

```diff
@@ -1,112 +1,99 @@
 <template>
   <div class="w-full p-4 border-b hover:bg-ligther flex">
-    <div class="flex-none mr-4">
-      <img :src="avatar" class="h-16 w-16 rounded-full flex-none" />
-    </div>
-    <div class="w-full">
-      <div class="flex flex-wrap items-center text-left w-full">
-        <button>
```

Comment:
sound like unecessary changes. please use prettier to format code as it was used in the repository.

Reply:
The changes in this file eliminates warnings.